package.json

```diff
@@ -1,68 +1,60 @@
+    "lint": "run-p lint:css lint:html lint:js lint:md",
+    "lint:css": "stylelint 'src/**/*.less' --fix",
+    "lint:html": "htmlhint 'src/**/*.html'",
+    "lint:js": "eslint --quiet --fix --format=pretty src",
+    "lint:md": "eslint --quiet --fix --format=pretty --ext .md .",
+    "prepare": "rollup -c",
+    "start": "rollup -c --watch",
+    "test": "yarn run lint",
+    "version": "npx conventional-changelog-cli -p angular -i CHANGELOG.md -s && git add CHANGELOG.md",
+    "postversion": "git push && git push --tags"
   },
-  "author": "OVH SAS",
-  "license": "BSD-3-Clause",
-  "main": "./dist/ovh-angular-chatbot.min.js",
   "dependencies": {
+    "ovh-ui-kit": "^2.23.0",
+    "simple-web-notification": "^1.0.31"
+  },
+  "devDependencies": {
+    "@commitlint/cli": "^7.1.2",
+    "@commitlint/config-angular": "^7.1.2",
+    "@ovh-ux/component-rollup-config": "^5.0.0-beta.1",
+    "eslint": "^5.9.0",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-formatter-pretty": "^2.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-markdown": "^1.0.0",
+    "htmlhint": "^0.10.1",
+    "husky": "^1.1.3",
+    "lint-staged": "^7.3.0",
+    "npm-run-all": "^4.1.5",
+    "stylelint": "^9.5.0",
+    "stylelint-config-standard": "^18.2.0"
+  },
+  "peerDependencies": {
     "angular": "^1.6.4",
     "angular-animate": "^1.4.0",
-    "angular-web-notification": "^1.2.22",
-    "debug": "^2.6.8",
     "jquery-ui": "^1.12.1",
     "ng-embed": "^2.2.0",
-    "ovh-angular-user-pref": "^0.3.1",
-    "ovh-ui-kit": "git+https://github.com/ovh-ux/ovh-ui-kit.git"
-  },
-  "devDependencies": {
-    "angular-mocks": "~1.6.x",
-    "autoprefixer-core": "^5.2.1",
-    "babel-eslint": "^7.2.3",
-    "babel-preset-es2015": "^6.24.1",
-    "cz-ovh-commit": "^1.0.0",
-    "eslint": "^4.1.1",
-    "eslint-config-ovh": "^0.1.1",
-    "grunt": "0.4.x",
-    "grunt-babel": "^6.0.0",
-    "grunt-bump": "~0.5.0",
-    "grunt-complexity": "~0.1.0",
-    "grunt-contrib-clean": "~0.6.0",
-    "grunt-contrib-concat": "~0.5.0",
-    "grunt-contrib-copy": "~0.7.0",
-    "grunt-contrib-csslint": "~0.4.0",
-    "grunt-contrib-cssmin": "~0.13.0",
-    "grunt-contrib-less": "^1.4.1",
-    "grunt-contrib-uglify": "~0.9.0",
-    "grunt-contrib-watch": "~0.6.1",
-    "grunt-eslint": "^20.0.0",
-    "grunt-karma": "~0.12.0",
-    "grunt-ng-annotate": "~0.10.0",
-    "grunt-ng-templates": "^1.0.0",
-    "grunt-ngdocs": "^0.2.5",
-    "grunt-postcss": "^0.6.0",
-    "jasmine-core": "^2.6.3",
-    "karma": "~1.7.0",
-    "karma-jasmine": "~1.1.0",
-    "karma-jasmine-html-reporter": "^0.2.2",
-    "karma-ng-html2js-preprocessor": "^1.0.0",
-    "karma-phantomjs-launcher": "~1.0.4",
-    "karma-spec-reporter": "0.0.31",
-    "less-plugin-remcalc": "0.0.1",
-    "matchdep": "^0.3.0",
-    "phantomjs": "^2.1.7"
-  },
-  "config": {
-    "commitizen": {
-      "path": "./node_modules/cz-ovh-commit"
-    }
+    "ovh-angular-user-pref": "^0.3.1"
```

Comment:
Could you try the latest prerelease of `ng-ovh-user-pref` instead? 

```sh
yarn remove ovh-angular-user-pref 
yarn add --peer @ovh-ux/ng-ovh-user-pref
```

ref: https://github.com/ovh-ux/ng-ovh-user-pref/tree/v1.0.0-beta.0

Thanks:)